src/Avalonia.Build.Tasks/GenerateAvaloniaResourcesTask.cs

```diff
@@ -127,7 +127,7 @@ private bool PreProcessXamlFiles(List<Source> sources)
 
         public bool Execute()
         {
-            Enum.TryParse<MessageImportance>(ReportImportance, out _reportImportance);
+            Enum.TryParse(ReportImportance, true, out _reportImportance);
```

Comment:
Noticed this minor issue causing #8971. Decided to not open another PR for this one liner.